test/test_proxy_tensor.py

```diff
@@ -1033,20 +1035,24 @@ def f(a, b, c, d, e):
             return x
 
         fx_g = _trace(f, 2, 4, 8, 16, 32)
-        self._assert_no_guards(fx_g, 1)
+        self.assertExpectedInline(fx_g.shape_env.format_guards(exclude_ivar=True), """\
+ - Eq(2*s0, s1)
```

Comment:
nit: it seems like get_guard_expr() includes the guards that match up ivars with vars (Eq(t0.size(0), s0)), and format_guards() doesn't (e.g. in here). Intentional?